libnopegl/src/pgcraft.c

```diff
@@ -1183,10 +1183,10 @@ static int filter_pipeline_elems(struct pgcraft *s,
 
 static int find_attribute(const struct pgcraft *s, int32_t location)
 {
-    const struct darray *attribute_array = &s->filtered_pipeline_info.desc.attributes;
-    const struct pipeline_attribute_desc *descs = ngli_darray_data(attribute_array);
-    for (int32_t i = 0; i < (int32_t)ngli_darray_count(attribute_array); i++) {
-        const struct pipeline_attribute_desc *desc = &descs[i];
+    const struct darray *info_array = &s->filtered_pipeline_info.desc.attributes;
```

Comment:
You're introducing this function earlier: I believe you could name it "info array" already before (even if the underlying struct changes, you're still in "pipeline_info.desc" so "info_array" was working before.

Reply:
Done.